test/cluster-template.json

```diff
@@ -1,5 +1,40 @@
 {
     "addResources": [
+        {
+            "uid": "cluster__<<CLUSTER_NAME>>",
+            "resourceString": "clusters",
+            "properties": {
+                "cpu": 8,
+                "kind": "Cluster",
+                "name": "<<CLUSTER_NAME>>",
+                "label": {
+                    "env": "dev",
+                    "name": "<<CLUSTER_NAME>>",
+                    "cloud": "Amazon",
+                    "vendor": "OpenShift",
+                    "clusterID": "abcd7696-da2f-46f5-8dc6-0357c4aacf5e",
+                    "openshiftVersion": "4.10.5",
+                    "cluster.open-cluster-management.io/clusterset": "auto-gitops-cluster-set",
+                    "feature.open-cluster-management.io/addon-work-manager": "available",
+                    "feature.open-cluster-management.io/addon-search-collector": "available",
+                    "feature.open-cluster-management.io/addon-application-manager": "available",
+                    "feature.open-cluster-management.io/addon-iam-policy-controller": "available",
+                    "feature.open-cluster-management.io/addon-cert-policy-controller": "available",
+                    "feature.open-cluster-management.io/addon-config-policy-controller": "available",
+                    "feature.open-cluster-management.io/addon-governance-policy-framework": "available"
+                },
+                "nodes": 1,
+                "memory": "32104404Ki",
+                "created": "2022-06-02T15:54:08Z",
+                "apigroup": "internal.open-cluster-management.io",
+                "consoleURL": "https://console-openshift-console.apps.sno-xxx-2xxxxxx-xxxxx.devxx.red-chesterfield.com",
```

Comment:
Can we make this more generic?
```suggestion
                "consoleURL": "https://console-openshift-console.apps.<<CLUSTER_NAME>>.basedomain.com",
```

Reply:
Done. Didn't commit suggestion as I don't know how to signoff to commits here.